docs/docs/dev_docs/sandbox/main.md

```diff
@@ -2,51 +2,376 @@
 
-# GETTING STARTED:
+Here we will walkthrough the process of retrieving the Sandbox, installing the client libraries and using it to deploy and use a fully private token contract on the Aztec network.
 
-Very quick, and to the point one-pager: https://github.com/AztecProtocol/aztec-packages/issues/1406
+## What do you need?
 
-# Questions to answer:
+- Node.Js >= v18
+- Docker and Docker Compose (Docker Desktop under WSL2 on windows)
 
-- What is the sandbox?
-  - High level, pithy outline.
-- What can you do with the sandbox?
-  - Deploy contracts
-  - Simulate function execution
-  - Execute private and public contracts
+That's it...
 
-- How do you download and install the sandbox?
-  - How do you do it:
-    - From up.aztec.network?
-    - From docker?
-    - From source?
+## Ok, so how do I try it out?
 
-- How do you interact with the sandbox?
-  - @aztec/cli
-    - What is the cli?
-    - When would someone prefer using the cli over aztec.js?
-    - We should also have a detailed aztec-cli section (see below)
-  - aztec.js
-    - What is aztec.js?
-    - When would someone prefer using aztec.js over the cli?
-    - We should also have a detailed aztec.js section (see [here](../dapps/main.md))
-  - RPC calls directly?
-    - How would an advanced user make rpc calls directly to the sandbox? Can they, even?
+Well, you can find instructions [at the website](https://up.aztec.network).
 
-- aztec/cli
-  - Refer directly to the npm package's explanations?
+Or you can just curl the site instead like this:
 
+`/bin/bash -c "$(curl -fsSL https://up.aztec.network)"`
 
-FOR INSTRUCTIONS FOR BUILDING A WALLET, WE SHOULD WRITE DOCS [HERE](../wallets/building_a_wallet.md)
+It will download and execute a script invoking docker compose with 2 containers:
 
-ERRORS:
-- Add any error explanations to [errors.md](./common_errors.md) (and break that file into multiple files if it's too unwieldy).
+- Anvil
+- Aztec Sandbox
 
+It will need to create servers on localhost ports 8545 (Anvil) and 8080 (Sandbox) so you will need to ensure nothing conflicts with this.
 
-- RPC calls directly
-  - Lower priority, but eventually we should answer all of the aztec.js questions here as well.
\ No newline at end of file
+Within a few seconds the Sandbox should be ready for use!
+
+<Image img={require("/img/sandbox.png")} />
+
+## Great, but I want to know more about it
+
+Aztec's Layer 2 network is a fully programmable combined private/public ZK rollup. To achieve this, the network contains the following primary components:
+
+- Archiver - Syncs with L1 and maintains a read-only repository of on-L1 state.
+- World State - The collection of Merkle Trees.
+- P2P - Currently just a locally stored collection of pending transactions.
+- Sequencer - Responsible for ordering transactions into the rollup, executing the rollup circuits and publishing blocks.
```

Comment:
I'd remove these from the quickstart, we can expand more on these in an architecture section. A dev here is probably just concerned with node/rpc/client.

Reply:
Yeah I did wonder about that myself.